frontend/src/features/admin-form/settings/components/PaymentSettingsSection/PaymentsUnsupportedMsg.tsx

```diff
@@ -1,21 +1,33 @@
 import { Flex, Text } from '@chakra-ui/react'
 
+import { FormResponseMode } from '~shared/types'
+
 import Link from '~components/Link'
 
 import { SettingsUnsupportedSvgr } from '~features/admin-form/settings/svgrs/SettingsUnsupportedSvgr'
 
 import { usePaymentGuideLink } from './queries'
 
-export const PaymentsUnsupportedMsg = (): JSX.Element => {
+type PaymentsUnsupportedMsgProps = {
+  responseMode?: FormResponseMode
+}
+
+export const PaymentsUnsupportedMsg = ({
+  responseMode,
+}: PaymentsUnsupportedMsgProps): JSX.Element => {
   const { data: paymentGuideLink } = usePaymentGuideLink()
+  const modeText =
+    responseMode === FormResponseMode.Email
+      ? 'email mode'
+      : 'multi-respondent forms'
   return (
     <Flex justify="center" flexDir="column" textAlign="center">
       <Text textStyle="h2" as="h2" color="primary.500" mb="1rem">
-        Payments are not available in Email mode
+        Payments are not available in {modeText}
```

Comment:
Similar with above, might make sense to list `Payments is only available in storage mode` to make this more maintainable. @staceytan1998